package.json

```diff
@@ -69,6 +70,8 @@
     "graphql": "^14.7.0 || ^15.8.0"
   },
   "dependencies": {
+    "@graphql-tools/schema": "^7.1.5",
+    "@graphql-tools/utils": "^7.1.5",
     "graphql-tag": "^2.12.6",
     "graphql-tools": "^4.0.8",
```

Comment:
Since you are now using every thing from @graphql-tools/schema and @graphql-tools/utils I think you can remove this package